packages/light-apps/src/AddAccount/Create.tsx

```diff
@@ -2,256 +2,208 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import FileSaver from 'file-saver';
+import { Keyring } from '@polkadot/ui-keyring';
 import { mnemonicGenerate, mnemonicToSeed, naclKeypairFromSeed } from '@polkadot/util-crypto';
-import React from 'react';
-import { RouteComponentProps } from 'react-router-dom';
 import { AppContext } from '@substrate/ui-common';
 import { AddressSummary, ErrorText, FadedText, Input, Margin, MnemonicSegment, NavButton, Stacked, StyledLinkButton, SubHeader, WrapperDiv, WithSpaceAround } from '@substrate/ui-components';
+import FileSaver from 'file-saver';
+import { Either, left, right } from 'fp-ts/lib/Either';
+import { none, Option, some } from 'fp-ts/lib/Option';
+import React, { useContext, useState } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
 
 interface Props extends RouteComponentProps { }
```

Comment:
nit: interface defintions together and type definitions together, each with a line space between them.